gitlab.go

```diff
@@ -117,6 +117,12 @@ func (t *ISOTime) UnmarshalJSON(data []byte) error {
 	return err
 }
 
+// implement String method for ISOTime
+func (j ISOTime) String() string {
```

Comment:
Again to satisfy linters, the receiver name should be consistent with other declarations. So in this case it should be `(t ISOTime)`.